ui/k-info-panel/show_trace_results.kytos

```diff
@@ -0,0 +1,119 @@
+<template>
+  <k-accordion>
+    <k-accordion-item title="Trace Results">
+      <k-context-panel v-if="is_empty" title="Empty Result: Verify Inputs" icon="gear">
+        </k-context-panel>
+          <div v-else class="data_table">
+            <table>
+              <thead>
+                <tr>
+                  <th rowspan="2">Trace</th>
+                  <th rowspan="2">DPID</th>
+                  <th rowspan="2">Time</th>
+                  <th rowspan="2">Type</th>
+                  <th rowspan="1" colspan="2">Input</th>
+                  <th rowspan="1" colspan="2">Output</th>
+                </tr>
+                <tr>
+                  <th>Port</th>
+                  <th>Vlan</th>
+                  <th>Port</th>
+                  <th>Vlan</th>                   
+                </tr>
+              </thead>
+              <tbody>
+                <tr v-for="(item, index) in trace_results">
+                  <td>{{index}}</td>
+                  <td>{{item.dpid}}</td>
```

Comment:
It'd be great to pontentially have the metadata `node_name` here, and same for the port with `port_name` when it exists.